cx_Freeze/freezer.py

```diff
@@ -318,157 +210,71 @@ def _FreezeExecutable(self, exe):
             os.chmod(target_path, mode | stat.S_IWUSR)
 
         # Copy icon
-        if exe.icon is not None:
-            if sys.platform == "win32":
-                try:
-                    winutil.AddIcon(target_path, exe.icon)
-                except RuntimeError as exc:
-                    if self.silent < 3:
-                        print("*** WARNING ***", exc)
-                except OSError as exc:
-                    if "\\WindowsApps\\" in sys.base_prefix:
-                        if self.silent < 3:
-                            print(
-                                "*** WARNING *** Because of restrictions on "
-                                "Microsoft Store apps, Python scripts may not "
-                                "have full write access to built executable.\n"
-                                "You will need to install the full installer.\n"
-                                "The following error was returned:"
-                            )
-                            print(exc)
-                    else:
-                        raise
-            else:
-                target_icon = os.path.join(
-                    self.targetdir, os.path.basename(exe.icon)
-                )
-                self._CopyFile(exe.icon, target_icon, copyDependentFiles=False)
+        target_lib_dir = os.path.join(self.targetdir, "lib")
+        self._CopyIcon(exe=exe, target_dir=target_lib_dir,
+                           target_path=target_path)
```

Comment:
_CopyIcon only uses the exe param (and  target_path on windows), so target_lib_dir can be removed.

Reply:
Agreed. Fixed. (Not sure what I was thinking there!)